pkg/api/types.go

```diff
@@ -232,7 +232,7 @@ type Config struct {
 
 	// BuildVolumes specifies a list of volumes to mount to container running the
 	// build.
-	BuildVolumes VolumeList
+	BuildVolumes []string
```

Comment:
When we used `VolumeList` we had [validation for file names](https://github.com/openshift/source-to-image/blob/4dd77215907d810f8fbc9c23dd8c7454f89131eb/pkg/api/types.go#L499-L522). With this change we don't have them anymore. Are we sure that it won't cause problem, especially security-related?